librecad/src/lib/information/rs_information.cpp

```diff
@@ -109,12 +110,12 @@ bool RS_Information::isTrimmable(RS_Entity* e1, RS_Entity* e2) {
                     e1->getParent()==e2->getParent()) {
 
                 // in the same polyline
-                RS_EntityContainer* pl = e1->getParent();
+                RS_Polyline* pl = (RS_Polyline *)e1->getParent();
```

Comment:
The cast is applied to the result, in order to be able to assign it to a polyline pointer. I need it to be polyline pointer and not just a generic entity container, because I need access to the isClosed method. However, it might well be a static cast as you said above - the object type is verified beforehand.